pmd-apex/src/test/java/net/sourceforge/pmd/lang/apex/rule/ApexXPathRuleTest.java

```diff
@@ -34,10 +34,15 @@ void testFileNameInXpath() {
 
     @Test
     void testBooleanExpressions() {
-        Report report = apex.executeRuleOnResource(makeXPath("//BooleanExpression[@Operator='&&']"),
+        Report report = apex.executeRuleOnResource(makeXPath("//BooleanExpression[@Op='&&']"),
                                                    "BooleanExpressions.cls");
         assertSize(report, 1);
     }
 
-
+    @Test
+    void testOldOperatorProperty() {
+        Report report = apex.executeRuleOnResource(makeXPath("//BooleanExpression[@Operator='&&']"),
```

Comment:
This test should actually fail now, as `getOperator` has been removed...

Update: yes - it doesn't find any nodes anymore. I think, the test can be removed, it's not necessary.

Reply:
Done (removed the test).